cni/pkg/plugin/cnieventclient.go

```diff
@@ -45,7 +45,7 @@ func buildClient(address, path string) CNIEventClient {
 				return net.Dial("unix", address)
 			},
 		},
-		Timeout: 1000 * time.Millisecond,
+		Timeout: 5 * time.Second,
```

Comment:
As long as we don't

- make this configurable
- understand that we shouldn't make this _too_ large, since we really _want_ to fail (relatively) fast and let k8s reschedule the failed pod versus preferring long stalls if we are encountering transient load

moving this around is fine.